app/controllers/planning_applications/consultee/emails_controller.rb

```diff
@@ -26,21 +26,36 @@ def create
         end
       end
 
+      def update
+        respond_to do |format|
+          if @consultation.update(permitted_params)
+            format.html do
+              redirect_to consultation_url
+            end
+          else
+            format.html { render :index }
+          end
+        end
+      end
+
       private
 
       def set_consultees
         @consultees = @consultation.consultees
       end
 
       def permitted_params
-        params.require(:consultation).permit(*consultation_params)
+        params.require(:consultation).permit(*consultation_params).tap do |ps|
+          ps[:consultees_not_required] = ps[:consultees_not_required] == "true"
```

Comment:
Why is this needed? The `Consultation` model should convert `"true"` to `true` when assigning the params